universe/Ship.cpp

```diff
@@ -590,6 +589,7 @@ void Ship::SetShipMetersToMax() {
         if (max_it == m_part_meters.end())
             continue;
 
+        max_it->second.SetCurrent(Meter::LARGE_VALUE);
```

Comment:
don't think this is needed. the point is to set all the persistent meters to a nonzero value, so later clamping will reduce them to the max. the max meters will be reset and set by effects.